config/owasp/suppressions.xml

```diff
@@ -30,4 +30,14 @@
     <notes>This vulnerability exists across azure identity and microsoft authentication libraries </notes>
     <cve>CVE-2024-35255</cve>
   </suppress>
+  <suppress>
```

Comment:
There is already an entry for the same CVE. You can just update the version number in the other entry and remove this one